src/vmm/benches/cpu_templates.rs

```diff
@@ -2,25 +2,44 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 // Benchmarking cases:
-//   * `CustomCpuTemplate` Deserialization
+//   * `CustomCpuTemplate` JSON deserialization
+//   * `CustomCpuTemplate` JSON serialization
 
+use std::mem::size_of_val;
 use std::path::Path;
 
 use criterion::{criterion_group, criterion_main, Criterion};
-use vmm::guest_config::templates::test_utils::TEST_TEMPLATE_JSON;
+use vmm::guest_config::templates::test_utils::{build_test_template, TEST_TEMPLATE_JSON};
 use vmm::guest_config::templates::CustomCpuTemplate;
 
+#[inline]
+pub fn bench_serialize_cpu_template(cpu_template: &CustomCpuTemplate) {
+    serde_json::to_string(cpu_template);
+}
+
 #[inline]
 pub fn bench_deserialize_cpu_template(cpu_template_str: &str) {
     serde_json::from_str::<CustomCpuTemplate>(cpu_template_str);
 }
 
 pub fn cpu_template_benchmark(c: &mut Criterion) {
-    println!("Template size: [{}] bytes.", TEST_TEMPLATE_JSON.len());
+    println!(
+        "Template size for deserialization: [{}] bytes.",
+        TEST_TEMPLATE_JSON.len()
+    );
+    let test_cpu_template = build_test_template();
+    println!(
+        "Template size for serialization: [{}] bytes.",
```

Comment:
So this is the size of the template in memory. Would it be a bit misleading to print the two values side by side, although they are different by nature?

Reply:
I can clarify the difference in the log statement a bit.

Reply:
Yes, that would be good.